### PR TITLE
tea: update to 63.1.0

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: editors/tea/Portfile [hunk 1]
```diff
@@ -61,8 +73,10 @@ variant qt5 conflicts qt4 description {Use Qt 5} {}
 
 if {[variant_isset qt4]} {
     PortGroup           qmake 1.0
+
```
Comment (pmetzger) [inline editors/tea/Portfile:76]: Why the extra carriage return?

File: editors/tea/Portfile [hunk 2]
```diff
@@ -61,8 +73,10 @@ variant qt5 conflicts qt4 description {Use Qt 5} {}
...
 }
 
 if {[variant_isset qt5]} {
     PortGroup           qmake5 1.0
+
```
Comment (pmetzger) [inline editors/tea/Portfile:81]: Why the extra carriage return?

File: editors/tea/Portfile (hunk 1 above)
Comment (barracuda156) [reply editors/tea/Portfile:76]: Port lint wants an empty line after PortGroup.

File: editors/tea/Portfile (hunk 2 above)
Comment (barracuda156) [reply editors/tea/Portfile:81]: I do not know specific rationale, but lint wants it.

File: editors/tea/Portfile (hunk 1 above)
Comment (pmetzger) [reply editors/tea/Portfile:76]: "port lint" demands lots of silly things; we need to fix it. It spews insane amounts of garbage for all rust ports for example, and this is another issue (it should not insist on a blank line in this case.)

File: editors/tea/Portfile (hunk 2 above)
Comment (ryandesign) [reply editors/tea/Portfile:81]: `port lint` is trying to encourage an orderly Portfile in which the set of PortSystem and optional PortGroup lines at the top of the Portfile are followed by a blank line. It was never intended for PortGroup directives to appear anywhere other than after the PortSystem line at the top of the Portfile, though we do now sometimes find ourselves doing that. In this Portfile I had decided that blank lines after these unusual mid-port PortGroup lines were not beneficial.

**Pull request merged**